frame/treasury/src/lib.rs

```diff
@@ -15,53 +15,69 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! > Made with *Substrate*, for *Polkadot*.
```

Comment:
updated according to the guidelines https://github.com/paritytech/substrate/blob/master/docs/DOCUMENTATION_GUIDELINES.md#pallet-crates